### PR TITLE
Replace IA_MULTI_VGT_PARAM_PIPED

### PR DESCRIPTION
**Pull request by AMD-dwang**
IA_MULTI_VGT_PARAM_PIPED was deprecated on new register format mode.

### CODE REVIEW

File: lgc/patch/RegisterMetadataBuilder.cpp [hunk 1]
```diff
@@ -1477,19 +1475,15 @@ void RegisterMetadataBuilder::setIaMultVgtParam() {
       usePrimitiveId = gsBuiltInUsage.primitiveId;
     }
 
-    if (isIaMultVgtParamPiped) {
-      auto iaMultVgtParamPiped =
-          getGraphicsRegNode()[Util::Abi::GraphicsRegisterMetadataKey::IaMultiVgtParamPiped].getMap(true);
-      iaMultVgtParamPiped[Util::Abi::IaMultiVgtParamPipedMetadataKey::SwitchOnEoi] = usePrimitiveId;
+    auto iaMultVgtParam = getGraphicsRegNode()[Util::Abi::GraphicsRegisterMetadataKey::IaMultiVgtParam].getMap(true);
+    iaMultVgtParam[Util::Abi::IaMultiVgtParamPipedMetadataKey::SwitchOnEoi] = usePrimitiveId;
+    if (m_isNggMode || (m_gfxIp.major == 10 && !m_isNggMode)) {
```
Comment (amdrexu) [inline lgc/patch/RegisterMetadataBuilder.cpp:1480]: Maybe we can move the metadata definition of `Util::Abi::GraphicsRegisterMetadataKey::IaMultiVgtParamPiped`.

**Comment by amdvlk-admin** [comment]
### Test summary for commit 13486e8d466821f6a045cf27e4e4f442c5af7fba
<details><summary>CTS tests (Failed: 0/138443)</summary><ul><li>Built with version 1.3.5.2</li>
<details open><summary>Ubuntu navi3x, Srdcvk</summary><ul>

<li>Passed: 35211/69228 (50.9%)</li>
<li>Failed: 0/69228 (0.0%)</li>
<li>Not Supported: 34017/69228 (49.1%)</li><li>Warnings: 0/69228 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu navi2x, Srdcvk</summary><ul>

<li>Passed: 35241/69215 (50.9%)</li>
<li>Failed: 0/69215 (0.0%)</li>
<li>Not Supported: 33974/69215 (49.1%)</li><li>Warnings: 0/69215 (0.0%)</li>
</ul></details>


</ul></details>

File: lgc/patch/RegisterMetadataBuilder.cpp (hunk 1 above)
Comment (AMD-dwang) [reply lgc/patch/RegisterMetadataBuilder.cpp:1480]: Fixed.

**Comment by amdvlk-admin** [comment]
### Test summary for commit ca2b49f556f1b91527891ef4703325400a89ffc0
<details><summary>CTS tests (Failed: 0/138443)</summary><ul><li>Built with version 1.3.5.2</li>
<details open><summary>Ubuntu navi3x, Srdcvk</summary><ul>

<li>Passed: 35211/69228 (50.9%)</li>
<li>Failed: 0/69228 (0.0%)</li>
<li>Not Supported: 34017/69228 (49.1%)</li><li>Warnings: 0/69228 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu navi2x, Srdcvk</summary><ul>

<li>Passed: 35241/69215 (50.9%)</li>
<li>Failed: 0/69215 (0.0%)</li>
<li>Not Supported: 33974/69215 (49.1%)</li><li>Warnings: 0/69215 (0.0%)</li>
</ul></details>


</ul></details>

**Review by nhaehnle — APPROVED** [review]

**Pull request merged**